hm_4/Task1(alphabet)/Program.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task1_alphabet_
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {   
+            Console.ForegroundColor = ConsoleColor.Yellow;
+            Console.WriteLine("Вывод  букв английского алфавита в обратном порядке");
+            Console.WriteLine("Убедитесь что включена !!!АНГЛИЙСКАЯ РАСКЛАДКА НА КЛАВИАТУРЕ!!!");
+            Console.Write("Введите первую букву английского языка a-z:  ");
+            char firstletter = Convert.ToChar(Console.ReadLine());
+            Console.Write("Введите вторую букву английского языка a-z:  ");
+            char secondletter = Convert.ToChar(Console.ReadLine());
+            int firstInt = (int)firstletter;
+            int secondInt = (int)secondletter;
+            if (97 <= firstInt & firstInt <= 122 & 97 <= secondInt & secondInt <= 122)
```

Comment:
I would suggest to define all magic numbers (such as 97, 122) as either const or local variable with appropriate logic name. 
Please keep in mind - good developer do not use some hard-coded strings, numbers in its code.